backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/ProcessDownVmCommand.java

```diff
@@ -155,7 +158,18 @@ protected void executeCommand() {
             Guid alternativeHostsList = vmHasDirectPassthroughDevices ? getVm().getDedicatedVmForVdsList().get(0) : null;
             refreshHostIfNeeded(hostId == null ? alternativeHostsList : hostId);
         }
-
+        VdsManager vdsManager = resourceManager.getVdsManager(getParameters().getHostId());
```

Comment:
let's make sure not to repeat the same mistake we did in the past and retrieve the VdsManager instance as we do in line 149 above to prevent instantiating VdsManager when the vm might already be removed from the database